pkgs/applications/misc/wordbook/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3
+, meson
+, ninja
+, pkg-config
+, glib
+, gtk4
+, libadwaita
+, librsvg
+, espeak-ng
+, gobject-introspection
+, wrapGAppsHook4
+, appstream-glib
+, desktop-file-utils
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "wordbook";
+  version = "unstable-2022-08-30";
+
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "fushinari";
+    repo = "Wordbook";
+    rev = "f72d9e748b7e182649dc18cefe27f93cf674e4b3";
+    hash = "sha256-ZL2ZgTezgH6wXxEgI2YAYOjX/DPb5irAAvyz7XGFl+U=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  nativeBuildInputs = [
    gobject-introspection
```

Seems to be a problems described in #56943, adding `gobject-introspection` to both `nativeBuildInputs` and `buildInputs` is a work around.

---

It works normal in Gnome, i don't know it will failed in KDE.